src/database.ts

```diff
@@ -1549,6 +1556,10 @@ class Database extends common.GrpcServiceObject {
       const [metadata] = await this.getMetadata(gaxOptions);
       this.databaseDialect = metadata.databaseDialect;
     }
+    if (cb) {
+      cb!(null, this.databaseDialect);
```

Comment:
At this point do we still need `cb!` given that we confirmed above that the callback is non-nil?

Reply:
Yeah, we don't need. Updated